spec/models/user_spec.rb

```diff
@@ -76,16 +76,18 @@
   end
 
   describe "#accessible_organizations" do
```

Comment:
Consider inlining this into the `Ability` since that's the only thing that uses it. 100% non blocking.